Formula/cpmtools.rb

```diff
@@ -11,8 +11,18 @@ class Cpmtools < Formula
     sha256 "b810122c220af6b36ab9316deec811adca68313d4371f8a0121239c40b94a015" => :mavericks
   end
 
+  option "with-libdsk", "Enable libdsk support"
+
+  depends_on "libdsk" => :build if build.with? "libdsk"
```

Comment:
Yes, I'd misinterpreted the docs there :) There's no need for libdsk at all if not building `--with-libdsk` and it seems `:optional` provides that already. Much simpler!